runtime/README.rst

```diff
@@ -70,16 +70,16 @@ The following table shows the available devices along with supported features:
      - ``Identity``, ``PauliX``, ``PauliY``, ``PauliZ``, ``Hadamard``, ``Hermitian``, ``Hamiltonian``, and Tensor Product of Observables
      - ``Identity``, ``PauliX``, ``PauliY``, ``PauliZ``, ``Hadamard``, ``Hermitian``, and Tensor Product of Observables
    * - Expectation Value
-     - All observables; Finite-shots not supported
-     - All observables; Finite-shots not supported
+     - All observables; Finite-shots supported
+     - All observables; Finite-shots supported
      - All observables; Finite-shots supported
    * - Variance
-     - All observables; Finite-shots not supported
-     - All observables; Finite-shots not supported
+     - All observables; Finite-shots supported
+     - All observables; Finite-shots supported
      - All observables; Finite-shots supported
    * - Probability
-     - Only for the computational basis on the supplied qubits; Finite-shots not supported
-     - Only for the computational basis on the supplied qubits; Finite-shots not supported
+     - Only for the computational basis on the supplied qubits; Finite-shots supported
+     - Only for the computational basis on the supplied qubits; Finite-shots supported
```

Comment:
Do we not support observables for probs now?

Reply:
Observable support for `probs`, `sample`, and `counts` is a different ticket (wasn't P0 for this quarter). But, Shuli and I had worked on this support too. This is now supported for all of Lightning sims, thanks to Shuli's hard work! We can tackle it next quarter as P0.

Reply:
Yeah I remember seeing it added to lightning. I guess it can be a different PR on the Catalyst if it's too much work.

Reply:
It's not too much work, but it requires changes into the Quantum dialect and lowering passes as well as the runtime. My plan for this ticket was helping Shuli to get a better understanding of the runtime :) I didn't want to implement this support originally but the situation changed in the last min.. hehe.

Reply:
Implementing the support is a half-day work, I can give it a try next week and I hope we don't run into any bugs.